Makefile

```diff
@@ -8,6 +8,8 @@ all:
 	
 project:
 	swift package generate-xcodeproj
+	# Optional: set the generated project's indentation settings.
```

Comment:
Let's add a comment here to explain how to install the gem. I think it's safe to assume people will have ruby installed, but `sudo gem install xcodeproj` might not be obvious to non-rubyists.